crates/eth-rpc/src/servers/net_rpc.rs

```diff
@@ -23,12 +22,10 @@ impl NetApiServer for NetRpc {
     }
 
     fn peer_count(&self) -> Result<PeerCount> {
-        // TODO: replace jsonrpsee::types::ErrorObject with Provider::Error once NetRpc uses Provider
-        Err(EthApiError::<jsonrpsee::types::ErrorObject>::MethodNotSupported("eth_peerCount".to_string()).into())
+        Ok(PEER_COUNT.clone())
     }
 
     fn listening(&self) -> Result<bool> {
-        // TODO: replace jsonrpsee::types::ErrorObject with Provider::Error once NetRpc uses Provider
-        Err(EthApiError::<jsonrpsee::types::ErrorObject>::MethodNotSupported("eth_listening".to_string()).into())
+        Ok(false)
```

Comment:
Can we add a comment why we're currently not listening?

Reply:
what should be documented for the comment? @bajpai244 @Eikix

Reply:
```suggestion
        // Kakarot RPC currently does not support peer-to-peer connections
        Ok(false)
```

Reply:
Done!